src/components/LogDetailCard/LogDetailCard.style.ts

```diff
@@ -0,0 +1,145 @@
+import styled from "styled-components";
+
+interface LogDetailCardThumbnailProps {
+  thumbnailURL: string;
+}
+
+interface LogDetailCardInfoProps {
+  isSettingLayout: boolean;
+}
+
+export const LogDetailCardContainer = styled.div`
+  display: flex;
+  width: 30rem;
+  height: 12rem;
+  border-radius: 0.8rem;
+  background-color: ${(props) => props.theme.white};
+  cursor: pointer;
+`;
+
+export const LogDetailCardThumbnail = styled.div<LogDetailCardThumbnailProps>`
+  width: 12rem;
+  height: 12rem;
+  background-image: ${(props) => `url(${props.thumbnailURL})`};
+  background-position: center;
+  background-size: cover;
+  border-top-left-radius: 0.8rem;
+  border-bottom-left-radius: 0.8rem;
+`;
+
+export const LogDetailCardInfo = styled.div<LogDetailCardInfoProps>`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: calc(100% - 12rem);
+  padding: 1.8rem 1rem 1rem;
+
+  .infoTitle {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    h3 {
+      width: ${(props) => (props.isSettingLayout ? "calc(100% - 15px)" : "100%")};
+      font-size: 1.6rem;
+      font-weight: 700;
+      white-space: normal;
+      display: -webkit-box;
+      -webkit-line-clamp: 1;
+      -webkit-box-orient: vertical;

```

Comment:
```suggestion
      white-space: nowrap;
     text-overflow : text-overflow: ellipsis;
```

[ P999 ] 여기는 한줄이라 위와 같이 처리할 수도 있을 것 같은데
 webkit 말고 요렇게는 적용하면 예상과 다르게 동작하실까요?!

Reply:
좋은 말씀이십니다 !
한 줄은 재영님이 말씀해 주신 코드로도 정상 동작할 것 같네요 !!
습관적으로 적어버렸네요..! ㅎㅎ 😂 

(일반 css 속성과 webkit css 속성의 차이에 대해서 알아보니 **브라우저 호환성 측면에서 webkit은 정말 필요할 때만 사용**하는 것이 좋다고 하네요..! 수정하겠습니다 감사합니다 ㅎㅎ)